src/scripts/modules/countdown-timer.js

```diff
@@ -0,0 +1,60 @@
+const init = () => {
+  Array.from(document.querySelectorAll('.countdownTimer'))
+    .forEach((timer) => {
+      let days = 0;
+      let hours = 0;
+      let minutes = 0;
+      let seconds = 0;
+      let count = 0;
+
+      const daysElement = timer.querySelector('.countdownTimer-days span');
+      const hoursElement = timer.querySelector('.countdownTimer-hours span');
+      const minutesElement = timer.querySelector('.countdownTimer-mins span');
+      const secondsElement = timer.querySelector('.countdownTimer-secs span');
+      const endDate = new Date(timer.dataset.date).getTime();
+
+      if (typeof (endDate) !== 'number') {
```

Comment:
If our linter doesn't complain, then the parentheses around `endDate` are unnecessary.